### DISCUSSION TITLE
University Introductions :wave:

### DISCUSSION THREAD

**Discussion by BekahHW** [deepgram/community#16] [General help]
Hello to all of our friends from Universities! We'd love to know more about you and what you're interested in. In the comment section below, tell us:

- Who are you?
- Where are you from (university, location, department)?
- What type of tech are you interested in?

**Comment by unpjuliachristmas** [comment]
Hi! Not sure if this tread is still alive, but I’ll take my chances. I am working at a university in Japan. I’m in the middle of a project and wondering if there is any data on how well DG works with non-native speakers of English, or accented English. Would really be grateful for anything you might have.
